### PR TITLE
Request for review #1 - Micro-Reddit

### PR DESCRIPTION
**Pull request by wickathou**
NOTE: Repo restarted due to complex conflicts on previous repo, worked along with @chirchir12 

Added:
- Visual interface for the app
- Added dependencies between models
- Added users, posts and comments views

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
```
Comment (wathigo) [inline app/models/user.rb:1]: **[REQUIRED]**
* User should have many comments.  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/building_with_active_record/user_has_many_comments.png) from Odin Project).

**Review by wathigo — CHANGES_REQUESTED** [review]
### Changes Required
Great work team :clap:
I have been able to create a user and and a post successfully. However,
I ran into some issues while creating a comment.
See the comment :point_down: 

**After implementing the changes, submit another request for review.**

**Happy Coding :computer: 
Simon Wathigo.**

File: app/models/user.rb (hunk 1 above)
Comment (wickathou) [reply app/models/user.rb:1]: We actually have many comments:

`class User < ApplicationRecord
  has_many :posts, dependent: :delete_all
  validates :name, presence: true
  validates :email, presence: true, uniqueness: true
  validates_format_of :email, with: /\A[^@\s]+@([^@\s]+\.)+[^@\s]+\z/
end`

You have to first create a user and then a post and then you can add comments

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Nice work! @wickathou and @chirchir12 :clap:

Please consider taking a second look at your user model, the `has_many :comments` association is actually missing. Please make the required change before you request another review

**Pull request merged**